### DISCUSSION TITLE
Restart FE component

### DISCUSSION THREAD

**Discussion by swapkh91** [linsoss/doris-operator#38] [Q&A] [answered]
hi @Al-assad 
how do I restart the FE component only?
If I edit the cluster yaml and change something in `fe` section, will that redeploy everything or just the FE?
There is a global variable which I set using SQL and now FE needs to restart for that variable to take effect.

**Comment by Al-assad** [comment, marked as answer]
If you modify the fe section in the DorisCluster yaml, only the FE in the cluster will be restarted.

Or If you want to restart FE without modifying the any FE configuration, plz refer to: https://linsoss.github.io/doris-operator/docs/maintian/restart-doris-cluster/

**Reply by swapkh91** [reply]
got it, thanks @Al-assad 
so to add annotations, I need to upgrade the operator version too
to do that do I `flux pull artifact` of the latest version and just re-apply the operator yaml?

**Reply by Al-assad** [reply]
yeah
